api_tests/base/test_serializers.py

```diff
@@ -310,6 +317,15 @@ def test_field_with_two_kwargs(self):
         field = data['relationships']['two_url_kwargs']['links']
         assert_in('/v2/nodes/{}/node_links/{}/'.format(node._id, node._id), field['related']['href'])
 
+    def test_field_with_two_filters(self):
+        req = make_drf_request_with_version(version='2.0')
+        project = factories.ProjectFactory()
+        node = factories.NodeFactory(parent=project)
+        data = self.BasicNodeSerializer(node, context={'request': req}).data['data']
+        field = data['relationships']['field_with_filters']['links']
+        assert_in(urllib.quote('filter[target]=hello', safe='?='), field['related']['href'])
```

Comment:
`AssertionError: 'filter[target]=hello' not found in 'http://localhost:8000/v2/nodes/7yy4n/?filter%5Btarget%5D=hello&filter%5Bwoop%5D=yea'`